src/layout/Foot.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Card, CardBody, Bullseye, Grid, GridItem, TextContent, Text, TextVariants } from '@patternfly/react-core';
+import { BookIcon, OutlinedEnvelopeIcon, ChatIcon } from '@patternfly/react-icons';
+
+const Foot = () => (
+    <Card>
+        <CardBody>
+            <Bullseye>
+                <TextContent>
+                    <Text component={ TextVariants.h1 }>
+                        Services &amp;&amp; Support
```

Comment:
Double `&amp;`

Reply:
I thought I saw two of them. Good catch!